.github/workflows/run-tests.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - '2.x'
+      - '2.x-**'
```

Comment:
This is worth checking in since it makes testing via github actions much easier for future changes. See https://irc.farmos.org/bot/log/farmOS/2020-11-21#T57794 for background. I can make a follow-up change to the development documentation to explain how to take advantage of this.

Reply:
I like this. Can we give it its own commit, though? It is not directly related to "Run phpunit without XDebug to speed up tests". (Keeping it in this PR as a second commit is fine.)

Reply:
Sure

Reply:
Made a separate commit and rebased this one on top of it, then force-pushed it to this PR branch.